client/my-sites/domains/domain-management/transfer/transfer-out/select-ips-tag.jsx

```diff
@@ -0,0 +1,220 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import request from 'superagent';
+import { find, isEmpty, startsWith, toUpper } from 'lodash';
+import { localize } from 'i18n-calypso';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Dialog from 'components/dialog';
+import SearchCard from 'components/search-card';
+import SectionHeader from 'components/section-header';
+import Suggestions from 'components/suggestions';
+import FormButton from 'components/forms/form-button';
+import { saveDomainIpsTag } from 'state/domains/transfer/actions';
+import { getGainingRegistrar, isTransferDone } from 'state/selectors';
+
+const debug = debugFactory( 'calypso:domains:select-ips-tag' );
+
+class SelectIpsTag extends Component {
+	static ipsTagListUrl = 'https://widgets.wp.com/domains/ips-tag-list.min.json';
+
+	state = {
+		currentQuery: '',
+		ipsTagInput: '',
+		ipsTagList: [],
+		selectedRegistrar: {},
+		showDialog: false,
+	};
+
+	componentWillMount() {
+		request.get( SelectIpsTag.ipsTagListUrl ).then( ( response ) => {
```

Comment:
I think `componentDidMount` is preferred to load data from a remote endpoint. 

Pretty sure `componentWillMount` is called on server side rendering as well.

https://reactjs.org/docs/react-component.html#componentdidmount

Reply:
Nice. TIL, thanks!